concept_r_python_cheatsheet.Rmd

```diff
@@ -0,0 +1,185 @@
+# draw graphs using python and R
+
+Shuyue Xu
+
+```{r, include=FALSE}
+knitr::opts_chunk$set(echo = TRUE)
+```
+## things that need to be done before fitting models
+##### 1.check unique of the values
+##### 2.check null values, consider to drop values or adding a missing indicator
+##### 3.normalize numerical features
+##### 4.handle categorical feature using Label encoding or OneHotencoding or other...
+
+<br/> 
+
+## Continuous Variables
+we always care about asymmetry, outliers, multimodality, gaps, heaping, errors
+<br/> 
+
+### Graphs that usually used for continuous Variables:
+#### 1.Histograms: R base code:hist(x, …)
```

Comment:
Please fix so hashtag headers are not followed by numbers since the hashtags will create numbered subheadings